src/transformState.js

```diff
@@ -5,7 +5,28 @@
  * @param {Object[]} actions
  */
 function transformState(state, actions) {
-  // write code here
+  for (const key of actions) {
```

Comment:
One more thing from previous comments - please rename `key` to `action` 🙂

Checklist: 

<img width="523" alt="Screenshot 2023-04-29 at 18 28 21" src="https://user-images.githubusercontent.com/32853034/235313344-092ace15-3ecb-477c-8379-d2eb33dea41a.png">